src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -39,6 +39,7 @@ void HBProcessor::process(const cpp2::HBReq& req) {
     if (!ActiveHostsMan::machineRegisted(kvstore_, host)) {
       LOG(INFO) << "Machine " << host << " is not registed";
       handleErrorCode(nebula::cpp2::ErrorCode::E_MACHINE_NOT_FOUND);
+      setLeaderInfo();
```

Comment:
Why not use handleLeaderChanged directly?

Reply:
because it is not leader change, to avoid confuse